registrations/test_tasks.py

```diff
@@ -168,64 +168,6 @@ def test_opt_in(self):
             },
         )
 
-    @responses.activate
-    @mock.patch("registrations.tasks.group_send")
-    def test_send_webhook(self, websocket):
-        """
-        Sends a webhook to the specified URL with the registration status
-        Also send the status over websocket
-        """
```

Comment:
I don't think that we want to remove these tests completely, since it is `Sends a webhook ... Also send the status over websocket`, we just want to remove the websocket things, we still want to test that the webhooks work.

Reply:
The test mock uses `group_send` which I have deleted, which then caused the test to fail. I'll assume that the next step would be to to replace the mock.patch, but I'm not sure with what

Reply:
Since `group_send` isn't being used anymore, we can just remove that mock from the test, and remove the assertions on that mock, and it then should pass, since nothing is calling that mock anymore